app/services/import_polygons.rb

```diff
@@ -45,16 +45,15 @@ def call
         end
         polygons.push(polygon)
       end
-      polygons_hash = { "polygons" => polygons }
 
       location_polygon = LocationPolygon.find_or_create_by(name: location_name)
 
       # Update location_type separately to reduce chance of creating duplicate LocationPolygons for the same location
       # when we change a location's location-type mapping.
       location_polygon.update(location_type: LOCATIONS_MAPPED_TO_HUMAN_FRIENDLY_TYPES[location_name])
 
-      # Skip buffers API call if the points and BUFFER_RADII have not changed since last time we used them to calculate the buffers.
-      location_polygon.update(polygons: polygons_hash, buffers: get_buffers(polygons)) if polygons_hash != location_polygon.polygons || location_polygon.buffers.keys.map(&:to_i) != BUFFER_RADII
+      # Skip buffers API call if the points and the BUFFER_RADII have not changed since last time we used them to calculate the buffers.
+      location_polygon.update(buffers: get_buffers(polygons)) if location_polygon.buffers&.keys&.map(&:to_i) != BUFFER_RADII
```

Comment:
I wonder should we still check that at least one of the buffered sets of polygons is different? At the moment this just checks that the radius options are different. We checked if the polygon itself had changed to make sure we picked up any bugfixes, e.g. if the polygon is in Wales when it should be in Liverpool.